### DISCUSSION TITLE
Welcome to RAG Discussions!

### DISCUSSION THREAD

**Discussion by JorjMcKie** [pymupdf/pymupdf4llm#14] [Announcements]
Please use this space for comments, questions, suggestions regarding the solutions provided in this repository.

We value your ideas and we will do our best to add more examples and to further develop [pymupdf4llm](https://pypi.org/project/pymupdf4llm/).

Thank you!

**Comment by Aidenren1215** [comment]
I'm using pymupdf for data extraction (text, table basically) in a RAG pipeline. Can give examples of how to extract tables properly using pymupdf? now the problems I encountered is tables in academic papers and financial report of companies can't be detected.
